instrumentation-core/spring/spring-webflux-5.0/README.md

```diff
@@ -0,0 +1,87 @@
+# Manual Instrumentation for Spring-WebFlux
+
+Provides OpenTelemetry instrumentation for Spring's WebClient.
+
+## Quickstart
+
+### Add these dependencies to your project.
+
+Replace `SPRING_VERSION` with the version of spring you're using.
+`Minimum version: 5.0`
+
+Replace `OPENTELEMETRY_VERSION` with the latest stable [release](https://mvnrepository.com/artifact/io.opentelemetry).
+`Minimum version: 0.8.0`
+
+For Maven add to your `pom.xml`:
+```xml
+<dependencies>
+  <!-- opentelemetry -->
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-spring-webflux-5.0</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+   <!-- provides opentelemetry-sdk -->
```

Comment:
i'm not sure we need to include note about providing the opentelemetry-sdk, i like the message of "use the OpenTelemetry API and/or use OpenTelemetry library instrumentation, then add an exporter, and you are good to go".

i think that's the message we agreed on in #846, but it would be good to get @anuraaga's thoughts here also

users don't need to care about the sdk, they just use API and add an exporter, and 
```suggestion
   <!-- opentelemetry exporter -->
```